src/components/TeamMembersList/index.tsx

```diff
@@ -0,0 +1,33 @@
+import List from 'components/List';
+import React, {useCallback} from 'react';
+import {ListItem, UserData} from 'types';
+
+interface TeamMembersListProps {
+    teamMembersData: UserData[];
+}
+
+const TeamMembersList = ({teamMembersData}: TeamMembersListProps) => {
```

Comment:
The idea here is very similar to what we are discussing on the topics in `List` component and `TeamLeadCard`. Basically, this implementation is improving reusability and changeability. Also, separating responsibilities and using component composition